### PR TITLE
doc: KNX IoT post-release amendments

### PR DESCRIPTION
**Pull request by annakielar**
Updated the files structure.
Changed plain text into steps where possible.
Updated the diagram's legend.

### CODE REVIEW

File: docs/about_knxiot.rst [hunk 1]
```diff
@@ -21,29 +24,31 @@ This chapter covers:
...
-    Moreover, if it is needed and there are enough hardware resources, this feature can be implemented as an additional service on Thread Border Router device.
 
-The diagram below presents a sample of the KNX installation that combines KNX devices using various communication methods within a single system.
+    Moreover, if it is needed, and there are enough hardware resources, this feature can be implemented as an additional service on the Thread Border Router device.
 
- .. figure:: /images/knx_iot_sys_sample.svg
-    :alt: Sample KNX installation consisting of KNX classic and IoT devices.
+The diagram below presents a sample of the KNX installation that combines KNX devices using various communication methods within a single system:
+
+.. figure:: /images/knx_iot_sys_sample.svg
```
Comment (annakielar) [inline docs/about_knxiot.rst:47]: The diagram's legend to be updated: what is the difference between solid and  dashed line? Is is a connection type, i.e., wireless vs. wired?

File: docs/environment_setup.rst [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+nRF Command Line Tools
+======================
+
+Download the nRF Command Line from the `nRF Command Line Tools`_ page.
+
+Wireshark
+=========
+
+Optionally, for tracing and sniffing purposes, install `Wireshark`_.
+To configure Wireshark for Thread packets sniffing, go to the `Configuring Wireshark for Thread`_ page.
```
Comment (annakielar) [inline docs/environment_setup.rst:60]: Infocenter is a private website. Should we link to any other public pages, or add guidelines directly here?

File: docs/about_knxiot.rst [hunk 3]
```diff
@@ -21,29 +24,31 @@ This chapter covers:
...
  .. note::
     The middleware functionality could be implemented as a separate device called KNX IoT router.
     It is connected to the same IP network as the KNX IP router.
 
     The KNX IoT router is responsible for transmitting data in both directions:
-    
-    * From KNX IoT Point API devices through the Thread Border Router to other KNX devices through the KNX IP router.
+
+    * From KNX IoT Point API devices through the Thread Border Router and KNX IP routers to other KNX devices.
     * From other KNX devices through the KNX IP, KNX IoT and Thread Border Routers to KNX IoT Point API devices.
```
[the remark this answers is not in the document]
Comment (milewr) [reply docs/about_knxiot.rst:41]: "KNX IP router" instead "KNX IP routers"

File: docs/about_knxiot.rst (hunk 1 above)
Comment (milewr) [reply docs/about_knxiot.rst:47]: Good point... solid line means wired connection, dashed line - wireless connection.

**Pull request merged**